src/server/engine_shard_set.cc

```diff
@@ -621,4 +692,18 @@ void EngineShardSet::TEST_EnableCacheMode() {
   RunBriefInParallel([](EngineShard* shard) { shard->db_slice().TEST_EnableCacheMode(); });
 }
 
+ShardId Shard(std::string_view v, ShardId shard_num) {
+  if (ClusterConfig::IsEnabledOrEmulated()) {
+    v = ClusterConfig::KeyTag(v);
```

Comment:
I suggest limiting this feature for now only to hashtags just because we do not know why we need it elsewhere.

Reply:
Done.